ava-labs/core-extension
# fix: allow reattaching forgotten ledger device

## Description
* [CP-9029](https://ava-labs.atlassian.net/browse/CP-9029)

## Testing
1. Remove your Ledger device for extension in Chrome settings: `chrome://settings/content/usbDevices`
2. Try to sign any transaction
3. You should be prompted to plug in your Ledger device, but it won't work.
4. Click `Unable to connect?`
5. In the new tab, click "Reconnect" and device selection should show up.
6. Once connected, your next transaction attempt should succeed.

## Screenshots:

Uploading Screen Recording 2024-08-02 at 22.41.58.mov…


## Checklist for the author
